drivers/SmartThings/philips-hue/src/disco.lua

```diff
@@ -348,11 +348,12 @@ process_discovered_light = function(driver, bridge_id, resource_id, device_info,
     }
 
     HueDiscovery.light_state_disco_cache[light.id] = {
+      hue_provided_name = light.metadata.name,
       id = light.id,
       on = light.on,
       color = light.color,
       dimming = light.dimming,
-      color_temp = light.color_temperature,
+      color_temperature = light.color_temperature,
```

Comment:
how confident are we that this is correctly referred to everywhere now?

Reply:
The only place it was being used incorrectly was in the `migrate_light` code path.